iphone/iphone/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -37,6 +47,16 @@
       "scale" : "3x"
     },
     {
+      "idiom" : "ipad",
+      "size" : "20x20",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "ipad",
+      "size" : "20x20",
+      "scale" : "2x"
+    },
+    {
```

Comment:
How is this change related to the PR? Please restore the initial Contents.json or give me a quick explanation on this. The icons are generated automatically for packaged SDK's anyway. :-)